platformminigbm.cpp

```diff
@@ -83,20 +88,42 @@ int DrmMinigbmImporter::ImportBuffer(buffer_handle_t handle, hwc_drm_bo_t *bo) {
   bo->width = gr_handle->width;
   bo->height = gr_handle->height;
   bo->hal_format = gr_handle->droid_format;
-  bo->format = gr_handle->format;
+  if (need_modifier)
+    bo->format = DRM_FORMAT_NV12;
+  else
+    bo->format = gr_handle->format;
```

Comment:
the code need move down together with 
if (need_modifier == false) {

Reply:
Done.